runtime/krest/src/lib.rs

```diff
@@ -955,6 +885,39 @@ impl peaq_pallet_storage::Config for Runtime {
 	type WeightInfo = peaq_pallet_storage::weights::SubstrateWeight<Runtime>;
 }
 
+// Zenlink-DEX Parameter definitions
```

Comment:
Hmmm, in the krest network, we are using chain ID, 2000 and 3000. We need to discuss about how to deal with the correct chain ID.

Reply:
Just like before, let us discuss it in the Asana task.